packages/mdc-slider/foundation.js

```diff
@@ -30,24 +32,38 @@ const KEY_IDS = {
   PAGE_DOWN: 'PageDown',
 };
 
-// Events that can constitute the user releasing drag on a slider
+/** @enum {string} */
+const EVENT_MAP = {
```

Comment:
I would suggest renaming this to something more descriptive like `MOVE_EVENT_MAP`.